Source/App/EncApp/EbAppConfig.c

```diff
@@ -122,24 +124,34 @@
 /**********************************
  * Set Cfg Functions
  **********************************/
-static void SetCfgInputFile                     (const char *value, EbConfig *cfg)
+static void SetCfgInputFile(const char *filename, EbConfig *cfg)
 {
-    if (cfg->input_file && cfg->input_file != stdin)
+    if (cfg->input_file && cfg->input_file_is_fifo)
         fclose(cfg->input_file);
-    if (!strcmp(value, "stdin"))
+
+    if (!filename) {
+        cfg->input_file = NULL;
+        return;
+    }
+
+    if (!strcmp(filename, "stdin"))
         cfg->input_file = stdin;
     else
-        FOPEN(cfg->input_file, value, "rb");
-    /* if input is a YUV4MPEG2 (y4m) file, read header and parse parameters */
-    if(cfg->input_file!=NULL){
-        if(check_if_y4m(cfg) == EB_TRUE)
-            cfg->y4m_input = EB_TRUE;
-        else
-            cfg->y4m_input = EB_FALSE;
-    }else{
-        cfg->y4m_input = EB_FALSE;
-    }
+        FOPEN(cfg->input_file, filename, "rb");
+
+#if defined(_MSC_VER)
```

Comment:
```suggestion
#ifdef _WIN32
```

`GetFileType` is also used for mingw-w64

Reply:
Commited to PR branch. Thanks.